pontoon/base/static/js/fluent_interface.js

```diff
@@ -103,8 +103,8 @@ var Pontoon = (function (my) {
       }
 
       // Render SelectExpression
-      if (element.type === 'SelectExpression') {
-        element.variants.forEach(function (item) {
+      if (element.expression && element.expression.variants) {
```

Comment:
Here as well, `variants` is always an non-empty array. Otherwise it's a syntax error.